sections/plate_notation.tex

```diff
@@ -0,0 +1,61 @@
+
+We replicate some layers in our \gls{dag} structure based on the structure from the taxonomy field within our dataset, having some nodes represent a topic based on a specific taxonomy.
+An example of this can be seen in \autoref{fig:pachinko_dag}, where we have the node "STEDER", and this is connected to "Danmark" in the third layer.
+To make the algorithm construct the topics to be based on our taxonomies, we introduce a novel locking mechanism for the Gibbs sampler which we use to run \gls{pam}.
+This mechanism is discussed further in \autoref{subsec:mod_pachinko}.
+
+We use a five-level\vejleder{hvorfor bruger vi en 5 lag} pachinko tree structure, following the format presented by \citet{li2006pachinko}.
+The first layer is the root layer which all topics are a part of.
+The last layer is the word layer consisting of one node for each word in the vocabulary of our corpus.
+The second and third layers will be constructed based on the entries of the first two positions in our taxonomy metadata, meaning there will be one node for each unique sub-taxonomy entry that is in the first or second position in the taxonomy sequence (e.g. "STEDER" and "Danmark", which is taken from "STEDER/Danmark/Aalborg", but not "Aalborg" since it is in the third position).
+We only use the first two layers for this, as these are among the most informative, and because introducing even more layers would slow down the training significantly, since the probability of all possible combinations of topics needs to be sampled for every word during training.
+From our experiments, the training time for 50 epochs, increases from 12 hours to 130 hours between four and five level pachinko.
+The fourth layer consists of $K = 90$ 'blank' topics, as with the other models we present in this paper.
+This layer is added so that the model can construct its own topics based on the higher-level topics learned from our taxonomy metadata.
+
+\input{figures/dag_structure}
+
+
+
+
+\subsubsection{Modification to Pachinko Allocation}\label{subsec:mod_pachinko}
+\vejleder{hvordan ? algoritmisk?}
+Normally when working with topic modeling, one does not know which topics will be present in the document set before training the model.
+However, the taxonomy metadata fields provide some general subject names of different levels of abstraction and some amount of documents attached to these subject names.
+This provides a unique opportunity for using the existing taxonomies as higher-level topics.
+Without modification, \gls{pam} will find topics with the same structure as our taxonomy, but the taxonomy values would be disregarded during training since it would generate new taxonomy sequences.
+However, in our case, we have a partially observed taxonomy and we want to use the existing meta-information to estimate the topics quicker and more accurately.
+Although, only ${\sim}25\%$ of the documents in our dataset have an observed taxonomy.
```

Comment:
Hvad gør vi med topic layers? er de fastsat for hver dokument som har en taxonomy og så vælger vi bare et topic i den sequence?